### DISCUSSION TITLE
Wallet Adapter example for vanilla js / non-react apps?

### DISCUSSION THREAD

**Discussion by nalin** [aptos-labs/aptos-developer-discussions#113] [Questions] [answered]
I know the [aptos-wallet-adapter](https://github.com/aptos-labs/aptos-wallet-adapter/tree/main) respository has the core package and the react package but I don't see any examples of how to use core directly or if it's even recommended. I'm building my app in sveltejs and am looking for any tutorials, examples, etc.

**Comment by 0xmaayan** [comment, marked as answer]
You are correct, the aptos-wallet-adapter has a core package that manages and handles the core logic and a React provider component to use in React applications.

We currently do not maintain other front end framework compatible components.

However, one can simply manage a provider component that works with the adapter core package, the implementation should be pretty straightforward  and can follow the React provider implementation https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-react/src/WalletProvider.tsx#L47

Note that the react provider package is an official Aptos package that is actively developed and maintained by Aptos and is up to date with latest updates and changes and therefore is recommended to use this package.

By the way, if you decided to build a sveltejs compatible package - open source it would be appreciated so others can contribute and use it also.
